pennylane/qchem/openfermion_obs.py

```diff
@@ -915,8 +900,72 @@ def molecular_hamiltonian(
     + (0.1676831945771896) [Z1 Z2]
     + (0.12293305056183801) [Z1 Z3]
     + (0.176276408043196) [Z2 Z3]
+
+    **New Function**
+    >>> symbols, coordinates = (['H', 'H'], np.array([0., 0., -0.66140414, 0., 0., 0.66140414]))
+    >>> molecule = qml.qchem.Molecule(symbols, coordinates)
+    >>> H, qubits = molecular_hamiltonian(molecule)
+    >>> print(qubits)
+    4
+    >>> print(H)
+    (-0.04207897647782188) [I0]
+    + (0.17771287465139934) [Z0]
+    + (0.1777128746513993) [Z1]
+    + (-0.24274280513140484) [Z2]
+    + (-0.24274280513140484) [Z3]
+    + (0.17059738328801055) [Z0 Z1]
+    + (0.04475014401535161) [Y0 X1 X2 Y3]
+    + (-0.04475014401535161) [Y0 Y1 X2 X3]
+    + (-0.04475014401535161) [X0 X1 Y2 Y3]
+    + (0.04475014401535161) [X0 Y1 Y2 X3]
+    + (0.12293305056183801) [Z0 Z2]
+    + (0.1676831945771896) [Z0 Z3]
+    + (0.1676831945771896) [Z1 Z2]
+    + (0.12293305056183801) [Z1 Z3]
+    + (0.176276408043196) [Z2 Z3]
     """
 
+    if len(molecule) == 1 and len(mol_prop) == 0:
+        if isinstance(molecule[0], qml.qchem.Molecule):
+            symbols = molecule[0].symbols
+            coordinates = molecule[0].coordinates
+            name = molecule[0].name
+            charge = molecule[0].charge
+            mult = molecule[0].mult
+            basis = molecule[0].basis_name
+            alpha = molecule[0].alpha
+            coeff = molecule[0].coeff
+        else:
+            ValueError("Only Molecule object is supported with this interface.")
```

Comment:
It should be `raise ValueError`, you can find examples of this in this function.